tensorflow/lite/kernels/internal/optimized/optimized_ops.h

```diff
@@ -2629,6 +2629,84 @@ inline void BroadcastMulDispatch(
                        input2_data, output_shape, output_data);
 }
 
+inline void Div(const ArithmeticParams& params,
+                const RuntimeShape& input1_shape, const float* input1_data,
+                const RuntimeShape& input2_shape, const float* input2_data,
+                const RuntimeShape& output_shape, float* output_data) {
+  gemmlowp::ScopedProfilingLabel label("Div");
+  const float output_activation_min = params.float_activation_min;
+  const float output_activation_max = params.float_activation_max;
+
+  int i = 0;
+  const int size = MatchingFlatSize(input1_shape, input2_shape, output_shape);
+#ifdef USE_NEON
+  static constexpr int kNewtonSteps = 2;
```

Comment:
Could you please add a comment on this parameter? How is it used and what is the accuracy implication if kNewtonStep is 2?

Reply:
Added a comment.

`kNewtonSteps` stands for a number of Newton-Raphson iterations used to refine the initial estimate of the reciprocal given by `vrecpeq_f32` instruction.

The list below shows implication of this parameter on maximum and average relative errors between the results of NEON multiplication by the reciprocal and the actual result of floating point division, where both of operands are random numbers in the range (2^-16, 2^16).

- kNewtonStep=0: max=2.76e-03 avg=9.70e-04
- kNewtonStep=1: max=7.57e-06 avg=1.39e-06
- kNewtonStep=2: max=1.52e-07 avg=3.49e-08
- kNewtonStep=3: max=1.19e-07 avg=2.52e-08